Dockerfile.prod

```diff
@@ -8,7 +8,7 @@ RUN yarn run build
 
 # production stage
 FROM nginx:stable-alpine as production-stage
-COPY --from=build-stage /app/dist /usr/share/nginx/html
-EXPOSE 80
+RUN sed  "s/80;/8080;/g" /etc/nginx/conf.d/*.confCOPY --from=build-stage /app/dist /usr/share/nginx/html
```

Comment:
Missing newline betwwen .conf and COPY ? @MichaelSp :)

Reply:
oh yeah...😊